appcelerator-archive/amp
# Image API and CLI command

Deployment on AWS can include a mirror registry, only available from inside the VPC.
This PR adds an image command to the CLI, allowing to list, push or remove images from the mirror registry with the AMP authorization model.

## Verification

- Deploy a cluster on AWS (https://github.com/appcelerator/amp/wiki/Deploy-a-dev-cluster-on-AWS), with the --aws-parameter MirrorRegistry=true
- check the amp image command: `amp -k -s REMOTE_URL image -h
- list the images (should be empty)
- push a local image with a name that doesn't exist on Docker Hub
- deploy on the cluster a service using this image

PS: to be rebased, some commits had to be added to this PR, since older PRs are waiting for review. only the last 3 commits are relevant to this PR.
